static/src/javascripts/components/enhancer/enhancer.js

```diff
@@ -1,8 +1,4 @@
-define([
-    'require'
-], function (
-    require
-) {
+define(function () {
```

Comment:
Maybe that's the clue. There's an inaccuracy in my comment, the module is dynamically required, but in an array. It actually reads `define(['require'], function(require){ require(['module'],cb) })`. Maybe it's the use of the special amd parameter in conjunction with arrays that cause a static require? Will investigate further.